### ISSUE TITLE
Issue running nvm use command

### ISSUE THREAD

**Issue by RikhiSingh** [usebruno/bruno#3612]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

When I run nvm dev it says it doesnt exist!

### Screenshots/Live demo link

It says something like nvm is not recognized by cmd

**Comment by ganesh-bruno** [comment]
Hey @RikhiSingh,

Could you please confirm whether you're able to set up Bruno locally?

The instructions in the contributing.md file should be clear for setting up Bruno. If you're encountering the issue with `nvm dev` saying it doesn't exist, it might be worth double-checking your `nvm` installation.

If you're still having trouble, would you mind sharing a screenshot or a quick video?

**Comment by RikhiSingh** [comment]
Yeah I got some help from youtube, apparently my system was missing nvm, so it could be installed using nvm install or from its official repo on GitHub.

I also added the same in contributing!
